nifi-api/src/main/java/org/apache/nifi/annotation/behavior/SupportsBatching.java

```diff
@@ -47,5 +47,5 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Inherited
 public @interface SupportsBatching {
-
+    RunDuration duration() default RunDuration.ZERO_MILLIS;
```

Comment:
We should probably call this `defaultDuration` or `defaultRunDuration`, rather than just `duration`. Can we also add some JavaDocs here to explain exactly what happens when this value is set?

Reply:
Yes, that's a good idea! Thanks.